src/controllers/submissionController.js

```diff
@@ -0,0 +1,85 @@
+const submissionQueries = require('../db/queries.submission.js');
+const mongoose = require('mongoose');
+const Mailgun = require('mailgun').Mailgun;
+const mg = new Mailgun(process.env.mgApi); //Need Mailgun API key
+
+module.exports = {
+  index(req, res, next) {
+    submissionQueries.getAllSubmissions()
+    .then((submissions) => {
+      let returnData = {
+        statusCode: 200,
+        message: 'Success',
+        data: submissions
+      };
+      res.json(returnData)
+    })
+    .catch((err) => {
+      console.log(err)
+      let returnData = {
+        statusCode: 400,
+        message: 'Bad Request',
+        data: err
+      };
+      res.json(returnData)
```

Comment:
We should set `res.status()` so that it doesn't return a `200` on error.

(this and the above comment go equally for the failure paths below, as well)

Reply:
Added.